library/Icingadb/ProvidedHook/Reporting/SlaReport.php

```diff
@@ -188,6 +188,12 @@ public function initConfigForm(Form $form)
             'min'         => '1',
             'max'         => '12'
         ]);
+
+        $form->addElement('checkbox', 'csv-json-export', [
+            'label'          => 'Export total for CSV and JSON',
+            'checkedValue'   => '1',
```

Comment:
Can you add a comment with a little explanation why we're setting `0` and `1` here?